### PR TITLE
chore(dropdowns): link custom filtering kb

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

**Review by dimodi — CHANGES_REQUESTED** [review]
Two adjacent links that point to the same location look strange. This setup may either trigger a "WTF" reaction or the reader can assume one of the links is wrong. Consider merging.

**Review by yordan-mitev — COMMENTED** [review]
Use title case in the anchor links, for example, "Custom Filtering by Multiple Fields".

File: components/autocomplete/filter.md [hunk 1]
```diff
@@ -87,3 +87,7 @@ By default, the filtering is debounced with 150ms. Configure that with the [`Deb
 ## See Also
 
 * [Live Demo: AutoComplete Filtering](https://demos.telerik.com/blazor-ui/autocomplete/filtering)
+
+* [Custom filtering by multiple fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/autocomplete/filter.md:91]:
```suggestion
* [Custom Filtering by Multiple Fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```

File: components/combobox/filter.md [hunk 2]
```diff
@@ -99,3 +99,7 @@ By default, the filtering is debounced with 150ms. Configure that with the [`Deb
 ## See Also
 
 * [Live Demo: ComboBox Filtering](https://demos.telerik.com/blazor-ui/combobox/filtering)
+
+* [Custom filtering by multiple fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/combobox/filter.md:103]:
```suggestion
* [Custom Filtering by Multiple Fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```

File: components/combobox/filter.md [hunk 3]
```diff
@@ -99,3 +99,7 @@ By default, the filtering is debounced with 150ms. Configure that with the [`Deb
...
+
+* [Filter by field that is not the TextField]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/combobox/filter.md:105]:
```suggestion
```

File: components/dropdownlist/filter.md [hunk 4]
```diff
@@ -103,3 +103,7 @@ By default, the filter input in the popup is empty. Set the desired hint in it t
 ## See Also
 
 * [Live Demo: DropDownList Filtering](https://demos.telerik.com/blazor-ui/dropdownlist/filtering)
+
+* [Custom filtering by multiple fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/dropdownlist/filter.md:107]:
```suggestion
* [Custom Filtering by Multiple Fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```

File: components/dropdownlist/filter.md [hunk 5]
```diff
@@ -103,3 +103,7 @@ By default, the filter input in the popup is empty. Set the desired hint in it t
...
+
+* [Filter by field that is not the TextField]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/dropdownlist/filter.md:109]:
```suggestion
```

File: components/multicolumncombobox/filter.md [hunk 6]
```diff
@@ -106,3 +106,7 @@ By default, the filtering is debounced with 150ms. Configure that with the [`Deb
 ## See Also
 
 * [Live Demo: MultiColumnComboBox Filtering](https://demos.telerik.com/blazor-ui/multicolumncombobox/filtering)
+
+* [Custom filtering by multiple fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/multicolumncombobox/filter.md:110]:
```suggestion
* [Custom Filtering by Multiple Fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```

File: components/multicolumncombobox/filter.md [hunk 7]
```diff
@@ -106,3 +106,7 @@ By default, the filtering is debounced with 150ms. Configure that with the [`Deb
...
+
+* [Filter by field that is not the TextField]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/multicolumncombobox/filter.md:112]:
```suggestion
```

File: components/multiselect/filter.md [hunk 8]
```diff
@@ -111,3 +111,7 @@ By default, the filtering is debounced with 150ms. Configure that with the [`Deb
 ## See Also
 
 * [Live Demo: MultiSelect Filtering](https://demos.telerik.com/blazor-ui/multiselect/filtering)
+
+* [Custom filtering by multiple fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/multiselect/filter.md:115]:
```suggestion
* [Custom Filtering by Multiple Fields]({%slug dropdowns-kb-search-in-multiple-fields%})
```

File: components/multiselect/filter.md [hunk 9]
```diff
@@ -111,3 +111,7 @@ By default, the filtering is debounced with 150ms. Configure that with the [`Deb
...
+
+* [Filter by field that is not the TextField]({%slug dropdowns-kb-search-in-multiple-fields%})
```
Comment (ntacheva) [inline components/multiselect/filter.md:117]:
```suggestion
```

**Review by dimodi — APPROVED** [review]

**Pull request merged**